ptu/global.ini

```diff
@@ -45311,19 +45311,19 @@ hud_FollowLeader2=Folgen
 hud_FollowLeader3=Folgen
 hud_FollowLeader4=Folgen
 hud_FollowLeader5=Folgen
-hud_FollowLeader6=Rennen Sie mit Gilly zum Spire
+hud_FollowLeader6=Rennen mit Gilly zum Turm
```

Comment:
Aus dem werde ich auch nicht Schlau

Reply:
Race Gilly To Spire -> "Fahre mit Gilly zum Turm um die Wette" (oder so ähnlich)

Reply:
Oder fliege

Reply:
Ein Rennen mit Gilly zum Turm. Da haben wir zwei Fliegen mit einer Klappe geschlagen. Juckt eh keinen

Reply:
Eigentlich sollte hier der imperativ verwendet werden. Aber wir können dann im Spiel das vlt sehen